src/MLJGLMInterface.jl

```diff
@@ -327,4 +327,301 @@ metadata_model(
     path = "$PKG.LinearCountRegressor"
 )
 
+"""
+$(MMI.doc_header(LinearRegressor))
+
+[`LinearRegressor`](https://en.wikipedia.org/wiki/Linear_regression) assumes the target is a continuous variable
+whose conditional distribution is normal with constant variance, and whose
+expected value is a linear combination of the features (identity link function).
+Options exist to specify an intercept or offset feature.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `scitype(X)`
```

Comment:
This is not wrong, but for users it's more convenient to check scitypes of a *table* using `schema(X)` (but for vector `y` they do use `scitype(y)` - so no change at 350). 

```suggestion
  are of scitype `Continuous`; check the scitype with `schema(X)`
```